test/tutorial/scripts/api/get_file_head_api.py

```diff
@@ -0,0 +1,12 @@
+from hca import HCAConfig
+from hca.dss import DSSClient
+
+hca_config = HCAConfig()
+hca_config["DSSClient"].swagger_url = f"https://dss.data.humancellatlas.org/v1/swagger.json"
+dss = DSSClient(config=hca_config)
+
+print(dss.head_file(uuid="6887bd52-8bea-47d9-bbd9-ff71e05faeee", replica="aws"))
```

Comment:
Please always expand and then print (don't just do this on one line) since this is supposed to be an explanatory tutorial and we want to make things clear.